pkg/block/azure/main_test.go

```diff
@@ -0,0 +1,100 @@
+package azure_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/service"
+	"github.com/ory/dockertest/v3"
+	"github.com/txn2/txeh"
+)
+
+const (
+	azuriteContainerTimeoutSeconds = 10 * 60 // 10 min
+	containerName                  = "container1"
+	accountName                    = "account1"
+	accountKey                     = "key1"
+)
+
+var (
+	pool     *dockertest.Pool
+	blockURL string
+)
+
+func runAzurite(dockerPool *dockertest.Pool) (string, func()) {
+	ctx := context.Background()
+	resource, err := dockerPool.Run("mcr.microsoft.com/azure-storage/azurite", "latest", []string{
```

Comment:
Pin a version or wake up one day to find out we're not compatible with `latest` no more :)